pai-management/README.md

```diff
@@ -273,7 +273,7 @@ sudo ./deploy.py -d -p /path/to/configuration/directory
 ## Cleanup your previous deployment
 
 ```
-sudo ./deploy.py -p /path/to/configuration/directory
+sudo ./deploy.py -p /path/to/configuration/directory -c
```

Comment:
1) -c option is designed for clean the generated template file.
2) Why we put an command ```sudo ./deploy.py -p /path/to/configuration/directory -c``` here is because when running the ./cleanup-service.py, some template file should be generated. So to prevent user to run ./cleanup-service directly, we recommand user to run the command firstly.

Reply:
OK, fine. I know what you means. The command `sudo ./deploy.py -p /path/to/configuration/directory` should be executed firstly to generate the template files which `./cleanup-service.py` need.